layers/meta-xt-dom0-gen4/recipes-kernel/linux-libc-headers/linux-libc-headers_%.bbappend

```diff
@@ -0,0 +1,4 @@
+# we need to avoid a situation when both linux and linux-libc-headers
+# are fetching the same sources simultaneously because this may result in
+# the rejection from the github
+do_fetch[depends] = "linux-generic-armv8:do_fetch"
```

Comment:
I am curious... can you use `virtual/kernel` instead of `linux-generic-armv8`?  This will allow us to skip hardcoded linux recipe name.

Reply:
Thanks! This is great advice!
Reworked.